src/librustdoc/clean/types.rs

```diff
@@ -2018,7 +2018,7 @@ crate enum GenericArg {
 #[derive(Clone, PartialEq, Eq, Debug, Hash)]
 crate enum GenericArgs {
     AngleBracketed { args: Vec<GenericArg>, bindings: Vec<TypeBinding> },
-    Parenthesized { inputs: Vec<Type>, output: Option<Type> },
+    Parenthesized { inputs: Vec<Type>, output: Option<Box<Type>> },
 }
```

Comment:
Can you add a `rustc_assert_size` so this doesn't grow again by accident? As we've seen, the perf bot won't catch it if it regresses.

Reply:
Sure! I was actually about to do that even before I saw this comment :)